index.js

```diff
@@ -0,0 +1,33 @@
+const firstName = document.getElementById("firstName");
+const lastName = document.getElementById("lastName");
+const age = document.getElementById("age");
+const phoneNumber = document.getElementById("phoneNumber");
+const male = document.getElementById("male");
+const female = document.getElementById("female");
+const errorMsg = document.getElementById("errorMsg");
+const submitButton = document.getElementById("submitButton");
+
+const validateAge = () => {
+  errorMsg.innerText =
+    age.value < 18 ? "Your Age should be \n greater or equal to 18" : "";
+};
+const validatePhoneNumber = () => {
+  errorMsg.innerText =
+    phoneNumber.value.length !== 10 ? "Number length should be 10" : "";
+};
+
+const buttonHandle = () => {
+  submitButton.innerText = "Submitted";
+}
+
+const onSubmit = (event) => {
+  event.preventDefault();
+  buttonHandle()
```

Comment:
@isagarjaiswal No need to create function **buttonHandle**, just put the code here only